src/components/molecules/GeoJsonForm.vue

```diff
@@ -0,0 +1,160 @@
+<script lang="ts">
+import { defineComponent, ref } from 'vue';
+import {
+  drawnShape, specifiedShape, geometryInputSelection,
+} from '@/store/search';
+import { useMap } from '@/store/search';
+
+export default defineComponent({
+  name: 'GeoJsonForm',
+
+  setup() {
+
+    const geoOptions = [
+      'Draw on Map',
+      'Copy/Paste GeoJson',
+      'Upload File',
+    ];
+    const geoJsonString = ref('');
+    const geoJsonErrorMessages = ref(['']);
```

Comment:
```suggestion
    const geoJsonErrorMessages = ref([]);
```

Does other code depend on this starting off with a single empty string as an element?